libs/utils/utils.js

```diff
@@ -395,7 +395,7 @@ export function appendHtmlToLink(link) {
   }
 }
 
-export const loadScript = (url, type) => new Promise((resolve, reject) => {
+export const loadScript = (url, type, mode) => new Promise((resolve, reject) => {
```

Comment:
Would have been better from the start to use an object, but we can at least start using one going forward rather than having this method with 5 params in the future
```suggestion
export const loadScript = (url, type, { mode } = {}) => new Promise((resolve, reject) => {
```